Libraries/src/net/relinc/libraries/application/StrainGauge.java

```diff
@@ -62,15 +96,24 @@ else if(description.equals(voltageCalibatedDescrip))
 
 	
 	public String stringForFile(){
-		String contents = "SPFX StrainGauge Verison:1" + SPSettings.lineSeperator;
-		contents += genericNameDescrip + ":" + genericName + SPSettings.lineSeperator;
-		contents += gaugeFactorDescrip + ":" + Double.toString(gaugeFactor) + SPSettings.lineSeperator;
-		contents += resistanceDescrip + ":" + Double.toString(resistance) + SPSettings.lineSeperator;
-		contents += shuntResistanceDescrip + ":" + Double.toString(shuntResistance) + SPSettings.lineSeperator;
-		contents += lengthDescrip + ":" + Double.toString(length) + SPSettings.lineSeperator;
-		contents += voltageCalibatedDescrip + ":" + Double.toString(voltageCalibrated) + SPSettings.lineSeperator;
-		
+
+
+		String contents = getJSONObject().toString();
 		return contents;
+
+	}
+	public JSONObject getJSONObject(){
```

Comment:
Let's get a better method name.. maybe `getPropertiesJSON` or something

and this could be `protected` instead of `public` as it's only used by child class?